test/e2e/e2e-test/src/test/java/org/apache/skywalking/e2e/storage/StorageE2E.java

```diff
@@ -156,6 +167,54 @@ void serviceInstanceTopo() throws Exception {
         verifyServiceInstanceRelationMetrics(topology.getCalls());
     }
 
+    @RetryableTest
+    void addUITemplate() throws Exception {
```

Comment:
The `template`-related operations should not be retryable, it should be successful or failed at one time. 

```suggestion
    @Test
    void addUITemplate() throws Exception {
```